src/test/scala/com/evolutiongaming/crypto/EncryptTestApp.scala

```diff
@@ -0,0 +1,13 @@
+package com.evolutiongaming.crypto
+
+object EncryptTestApp extends App {
```

Comment:
I'd call it Encrypt and make Decrypt an App as well, thus you can use it on command line